### DISCUSSION TITLE
Supporter-Certificate-Prices +50% | repeatedly redesigned by increase | Fairness and Transparency ...

### DISCUSSION THREAD

**Discussion by PARAViRTUALiZATiON** [sandboxie-plus/Sandboxie#3572] [Polls]
This is intended to open a completely factual, non-judgmental dialogue about the transparency of supporter certificate pricing:

Just about a few days ago, prices for nearly every type of certificate were [**50%**](https://web.archive.org/web/20240110033959/https://xanasoft.com/product-category/sandboxie/) lower.

These additional costs affect private users in many ways, especially because they generally have less financial flexibility (or wealth base) than companies or business users.

This time, there has been no specific redesign of the feature set to offset these particularly high additional costs in a different form, which makes the increase in price restructuring appear opaque.

This raises various questions in the areas of fairness and transparency - because:

* Is such a high additional burden really necessary, and is the distribution of the burden between private and business users really balanced?

* If a computer user recently purchased the same certificate, they are paying [**50%**](https://web.archive.org/web/20240110033959/https://xanasoft.com/product-category/sandboxie/) less for absolutely the same without any specific background, why is that?

(**Please also note that this is a machine translation**)

**Comment by PARAViRTUALiZATiON** [comment]
At first, my interest in a support certificate was also piqued, but it was diminished by the constantly relatively unexpected and mostly short-term changes, as previously in the case of the [separate appreciation certificate](http://web.archive.org/web/20230827194509/https://xanasoft.com/product/sandboxie-plus-advanced-upgrade/) for encryption - which cames additionally and goes up also by 50%.


With the most recent change:
Imagine, for example, a sudden **50%-increase** in the price of a device or a food item - without a similarly significant increase in content.

It should be beyond question that this is a great open-source community project that one would like to support for its preservation.

Perhaps this thread will shed some light on the situation in the future?
* _Best Regards_

(**Please also note that this is a machine translation**)

**Reply by bastik-1001** [reply]
I understand your point for low income households or users in different regions where the price point is of great concern, however Sandboxie still has its uses, without there being money spend on those features that require a certificate. I hope David can indeed shine some more light onto the intended use of the funds.

Sandboxie does not stop to function with the advanced features enabled once the year is over, so that it is not necessarily required to purchase a new certificate each year. I am not here to defend or explain the prices, since those are set by David, nor do I benefit from the commercial success of Sandboxie. The only thing that benefits me (and a lot of other people) is, that David is kept motivated to continue to work on Sandboxie.
